packages/fela-bindings/src/__tests__/__snapshots__/createComponentFactory-test.js.snap

```diff
@@ -889,6 +889,48 @@ Array [
 ]
 `;
 
+exports[`Creating Components with a Proxy for props from Fela rules should replace rare unallowed symbols in className with underscore 1`] = `
+Array [
+  "<style>
+    .rule_1_______________abrv9k {
```

Comment:
Unique, but unreadable class name, because most of the displayName contains unallowed symbols (`displayName = '1!@#$%^&*{}/=\\'`)